docs/installation.md

```diff
@@ -119,6 +119,7 @@ Configuration keys:
 |`toggleCommitStatus`| Map of strings, allow (non-repo-admin) users to change the [Github commit status](https://docs.github.com/en/rest/commits/statuses) state(from failure to success and back). This can be used to continue promotion of a change that doesn't pass repo checks. the keys are strings commented in the PRs, values are [Github commit status context](https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28#create-a-commit-status) to be overridden|
 |`commentArgocdDiffonPR`| Uses ArgoCD API to calculate expected changes to k8s state and comment the resulting "diff" as comment in the PR. Requires ARGOCD_* environment variables, see below. |
 |`autoMergeNoDiffPRs`| if true, Telefonistka will **merge** promotion PRs that are not expected to change the target clusters. Requires `commentArgocdDiffonPR` and possibly `autoApprovePromotionPrs`(depending on repo branch protection rules)|
+|`usaSHALabelForArgoDicovery`| The Default method for discovering relevant ArgoCD applications(for a PR) rely on fetching all appliation in repo and checking the `argocd.argoproj.io/manifest-generate-paths` **annotation**, this might cause perfromance issue on repo with large number of ArgoCD applications. The alternative is to add SHA1 of the application path as a  **label** and relay on ArgoCD server side filtering, label name is `telefonistka.io/component-path-sha1`.|
```

Comment:
nitpick:

```suggestion
|`useSHALabelForArgoDicovery`| The default method for discovering relevant ArgoCD applications (for a PR) relies on fetching all applications in the repo and checking the `argocd.argoproj.io/manifest-generate-paths` **annotation**, this might cause a performance issue on a repo with a large number of ArgoCD applications. The alternative is to add SHA1 of the application path as a  **label** and rely on ArgoCD server-side filtering, label name is `telefonistka.io/component-path-sha1`.|
```